src/main/java/catan/domain/model/user/UserBean.java

```diff
@@ -66,6 +76,27 @@ public void setGuest(boolean guest) {
         this.guest = guest;
     }
 
+    public String getLanguage() {
+        return language;
+    }
+
+    public void setLanguage(String language) {
+        this.language = language;
+    }
+
+    public String getCountry() {
+        return country;
+    }
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    public ResourceBundle getMsgs() {
```

Comment:
according to logic, it appears that GameUser is owner of the messages, but it is not true, I would remove this message, as it is just a a bean that is mapped to database table,  it is not service that can provide some messages